src/modules/iotjs_module_adc.c

```diff
@@ -216,16 +198,32 @@ static void iotjs_adc_close_worker(uv_work_t* work_req) {
 
 JHANDLER_FUNCTION(AdcConstructor) {
   DJHANDLER_CHECK_THIS(object);
-  DJHANDLER_CHECK_ARGS(2, object, function);
```

Comment:
This gives an error Internal Error (AdcConstructor), which in my opinion does not say why the error occured. I would rather stay with my jcallback check and error message